tests/test_urls.py

```diff
@@ -107,3 +107,47 @@ def test_something_else():
 
     result = django_pytester.runpytest_subprocess()
     assert result.ret == 0
+
+
+@pytest.mark.django_project(
+    extra_settings="""
+    ROOT_URLCONF = "empty"
+    """
+)
+def test_urls_concurrent(django_pytester: DjangoPytester) -> None:
```

Comment:
I copied this test over to my `main` branch, and ran this on py3.13

```shell
count=0
while true; do
    count=$((count + 1))
    if ! pytest tests/test_urls.py -vv > /dev/null 2>&1; then
        echo "Test failed on iteration $count"
        break
    fi
    printf "."
done
```

it has not failed -- keeps going, after 100+ iterations (keep in mind, i only copied the test, not the "fix")

Reply:
PS. I also ran this with `pytest -n auto` (same loop as above) -- no difference, i dont see the race condition